spec.html

```diff
@@ -40609,7 +40704,8 @@ <h1>Promise Resolve Functions</h1>
             1. Let _thenAction_ be _then_.[[Value]].
             1. If IsCallable(_thenAction_) is *false*, then
               1. Return FulfillPromise(_promise_, _resolution_).
-            1. Let _job_ be NewPromiseResolveThenableJob(_promise_, _resolution_, _thenAction_).
+            1. Let _thenJobCallback_ be HostMakeJobCallback(_thenAction_).
+            1. Let _job_ be NewPromiseResolveThenableJob(_promise_, _resolution_, _thenJobCallback_).
```

Comment:
`NewPromiseResolveThenableJob` now has a bug: it calls `GetFunctionRealm(_then_)`, but `_then_` has a become a JobCallback Record rather than function object.

Reply:
Will fix.